portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,38 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+import java.util.ArrayList;
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Juan!");
+    ArrayList<Comment> jsonarray = new ArrayList<Comment>();
+    Comment newcomment1 = new Comment("I love this site.");
+    Comment newcomment2 = new Comment("This is so cool.");
+    Comment newcomment3 = new Comment("All the cool kids are coding nowadays.");
+    jsonarray.add(newcomment1);
+    jsonarray.add(newcomment2);
+    jsonarray.add(newcomment3);
+
+    String json = convertToJson(jsonarray);
```

Comment:
```suggestion
    ArrayList<Comment> jsonArray = new ArrayList<Comment>();
    Comment newComment1 = new Comment("I love this site.");
    Comment newComment2 = new Comment("This is so cool.");
    Comment newComment3 = new Comment("All the cool kids are coding nowadays.");
    jsonArray.add(newComment1);
    jsonArray.add(newComment2);
    jsonArray.add(newComment3);

    String json = convertToJson(jsonArray);
```

nit: variable names should be in lowerCamelCase in Java